src/main/java/de/tum/in/www1/artemis/domain/metis/Posting.java

```diff
@@ -47,6 +46,11 @@ public abstract class Posting extends DomainObject {
     @Column(name = "tokenized_content")
     private String tokenizedContent;
 
+    @Transient
+    @JsonSerialize
+    @JsonDeserialize
```

Comment:
Replacing both of these annotations with `@JsonProperty` should also work.

Reply:
Nice one, thanks!